src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -112,20 +76,26 @@ def __init__(
             )
             bulk_logger.info(f"Set start method to default {multiprocessing.get_start_method()}.")
             multiprocessing_start_method = multiprocessing.get_start_method()
-        use_fork = multiprocessing_start_method in ["fork", "forkserver"]
-        self._flow_file = flow_executor._flow_file
-        self._connections = flow_executor._connections
-        self._working_dir = flow_executor._working_dir
-        self._use_fork = use_fork
+        self._use_fork = multiprocessing_start_method in ["fork", "forkserver"]
+
+        # Initialize some fields from inputs.
```

Comment:
Maybe _batch_timeout_sec/_line_timeout_sec  is not from inputs?

Reply:
The inputs here refers to the inputs of the init method. I will update it to make clear.